frontend/src/ui-components/AcmCountCard/AcmCountCard.tsx

```diff
@@ -215,7 +217,10 @@ export const AcmCountCard = (props: AcmCountCardProps) => {
             </Icon>
           )}
           <CardTitle>
-            {cardHeader.title} {props.alert}
+            <Flex rowGap={{ default: 'rowGapNone' }}>
+              <FlexItem spacer={{ default: 'spacerSm' }}>{cardHeader.title}</FlexItem>{' '}
```

Comment:
Why is there a space between the flex items? Also, I just noticed that the formatting broke on these cards with your first PR for this feature. The counts are no longer aligned to the bottom of the card.
<img width="5104" height="2568" alt="image" src="https://github.com/user-attachments/assets/31a2a813-91f6-499a-9a08-cfdde9d66d17" />

Reply:
Ack-- found the culprit...not sure why I did this: https://github.com/stolostron/console/pull/4992/files#diff-1bed80d2a8092baa46a1767b2a64af9b0a18d25148b0dc529aec98616bfbc1f5L110